docs/index.md

```diff
@@ -1,7 +1,15 @@
-<a href="https://islandora.ca">Islandora</a> is an open-source repository framework, that combines a Drupal front-end
-with a Fedora repository. It is a solution for institutions who want an extremely flexible and configurable preservation repository without creating a lot of custom code.
+[Islandora](https://islandora.ca) is an open-source framework that provides the necessary tools turn a [Drupal 8](https://www.drupal.org) website into a fully-functional preservation repository without the need for additional code, while remanining flexible enough to be extended and customized.
 
-Islandora 8 integrates [Drupal](https://www.drupal.org/) (currently, Drupal 8) and Duraspace's [Fedora Repository](https://wiki.duraspace.org/display/FF/Fedora+Repository+Home) project (currently, Fedora 5).
+Out of the box, this includes:
 
-Islandora 8 allows you to create nodes, media, files, and taxonomy terms in Drupal, which are converted to RDF
-and pushed into a Fedora repository. It also allows you to connect repository content with microservices, small webapps that transform files to other types for display or preservation, or to extract additional metadata.
+- A suite of customized Drupal resource types and configurations, packaged as the [Islandora 8 Drupal module](https://github.com/Islandora/islandora), that streamline the process of creating digital preservation resources natively within Drupal as nodes, media, files, and taxonomy terms
+- Complete integration between Drupal resources and Duraspace's [Fedora 5.x Repository](https://wiki.duraspace.org/display/FF/Fedora+Repository+Home), including RDF conversion and asset storage
+- The [Crayfish](https://github.com/Islandora/crayfish) suite of microservices that provides resource mapping to Fedora and automated generation of derivative images and OCR, and which can be extended to provide even more services
```

Comment:
We also generate video and audio, too.  Maybe just say `automated generation of derivative files` instead?

Reply:
True, forgot about that; will add